extensions/sql-database-projects/src/models/project.ts

```diff
@@ -266,6 +266,11 @@ export class Project {
 	public async addFolderItem(relativeFolderPath: string): Promise<FileProjectEntry> {
 		const absoluteFolderPath = path.join(this.projectFolderPath, relativeFolderPath);
 
+		// check if folder already has been added to sqlproj
+		if (this.files.find(f => f.relativePath === relativeFolderPath)) {
```

Comment:
how about casing?

Reply:
Windows and macOS are both case-insensitive, Linux isn't (by default)... It's probably fine to .toLowerCase() everything anyway, though.